### DISCUSSION TITLE
Vercel Logs - console.log() output

### DISCUSSION THREAD

**Discussion by Dinozaver959** [vercel/community#5500] [Help]
### Summary

Until just a few days ago I was able to see console output in the Vercel logs, which was great for debugging issues quickly for customers. However, with the latest change, I can no longer see any console outputs. 

Where can I find them now?

### Steps to Reproduce

just look at Vercel logs

**Comment by bbirec** [comment]
Same here.

When printing with console.log, some of them appear, and sometimes they don't appear at all.

**Comment by Ilaay** [comment]
Same, don't see them at all

[a comment by an automated account is not shown]
